canonical/ubuntu.com
# WD-4643 - RFQ: Build of the automotive buyers guide

## Done

- Built an interactive page at /automotive/buyers-guide

## QA
1. Open the demo link below
2. Check that this page matched [the design](https://www.figma.com/file/mQnuVCHip6j8Vlecqgqe2C/RFQ-engage-Automotive?node-id=0%3A1&mode=dev) and [the copy doc](https://docs.google.com/document/d/1NLnoNEP6cL46PotNGVuKLajehl-nfyBU4C8JT-FYMvQ/edit)

Fixes https://warthogs.atlassian.net/browse/WD-4643
